static/swagger-specs/unified-tags.yaml

```diff
@@ -0,0 +1,983 @@
+        - $ref: '#/components/parameters/content-type'
+        - $ref: '#/components/parameters/x-gw-ims-org-id'
+        - $ref: '#/components/parameters/x-api-key'
+      responses: 
+        200:
+          x-summary: "Success"
+          description: "A list of tags that belong to the organization is returned."
+          content: 
+            application/json:
+              schema: 
+                $ref: "#/components/schemas/tagsList"
+    post: 
+      tags: 
+        - Tags
+      summary: Create a new tag
+      operationId: createTag
+      parameters: 
+        - $ref: '#/components/parameters/authorization'
+        - $ref: '#/components/parameters/content-type'
+        - $ref: '#/components/parameters/x-gw-ims-org-id'
+        - $ref: '#/components/parameters/x-api-key'
+      requestBody:
+        description: The request body to create a tag. 
+        content: 
+          "*/*":
+            schema: 
+              $ref: "#/components/schemas/createTagBody"
+        required: true
+      responses: 
+        201:
+          x-summary: "Created"
+          description: "A new tag was successfully created."
+          content: 
+            application/json:
+              schema: 
+                $ref: "#/components/schemas/tag"
+  /tags/{TAG_ID}:
+    get:
+      tags:
+        - Tags
+      summary: Retrieve a tag
+      operationId: retrieveTag
+      parameters: 
+        - $ref: "#/components/parameters/tag-id"
+        - $ref: '#/components/parameters/authorization'
+        - $ref: '#/components/parameters/content-type'
+        - $ref: '#/components/parameters/x-gw-ims-org-id'
+        - $ref: '#/components/parameters/x-api-key'
+      responses: 
+        200:
+          x-summary: "Success"
+          description: "The tag was successfully retrieved."
+          content:
+            application/json:
+              schema: 
+                $ref: "#/components/schemas/tag"
+    patch: 
+      tags: 
+        -  Tags
+      summary: Update a tag
+      operationId: updateTag
+      parameters: 
+        - $ref: "#/components/parameters/tag-id"
+        - $ref: '#/components/parameters/authorization'
+        - $ref: '#/components/parameters/content-type'
+        - $ref: '#/components/parameters/x-gw-ims-org-id'
+        - $ref: '#/components/parameters/x-api-key'
+      requestBody:
+        description: "The request body to update the tag."
+        content: 
+          "*/*":
+            schema: 
+              $ref: "#/components/schemas/updateTagBody"
+        required: true
+      responses: 
+        200:
+          x-summary: "Success"
+          description: "The tag was successfully updated."
+          content:
+            application/json:
+              schema: 
+                $ref: "#/components/schemas/updatedTag"
+    delete: 
+      tags: 
+        - Tags
+      summary: Delete a tag
+      operationId: deleteTag
+      parameters: 
+        - $ref: "#/components/parameters/tag-id"
```

Comment:
You can't use this reference as it says : "The ID of the tag you're retrieving."
when really it should be 
"The ID of the tag you're deleting."

Reply:
I can't seem to overwrite the parameter description, so I've just kept it more vague.